api/src/main/java/com/debugmafia/clueless/model/Player.java

```diff
@@ -0,0 +1,52 @@
+package com.debugmafia.clueless.model;
+
+import java.util.Set;
+
+public class Player {
+
+  private String userName;
+  private int playerNum;
+  private Piece character;
+  private boolean isInControl;
+  private Set<Card> cardsInHand;
+
+  public Player() {
+  }
```

Comment:
I think we can remove this constructor. You shouldn't be able to create a `Player` if you don't give a `userName` and `Piece` at a minimum.